pkg/exchange/ftx/convert.go

```diff
@@ -129,3 +129,19 @@ func toGlobalTrade(f fill) (types.Trade, error) {
 		IsIsolated:    false,
 	}, nil
 }
+
+func toGlobalKLine(symbol string, interval types.Interval, h Candle) (types.KLine, error) {
+	return types.KLine{
+		Exchange:  types.ExchangeFTX.String(),
+		Symbol:    symbol,
```

Comment:
ToGlobalSymbol? because these klines will be stored in the db

Reply:
How to convert to global symbol, such as btc-perp?

Reply:
perp is for future, we don't support futures for now. will need the design for futures

Reply:
simply convert BTC/USDT to BTCUSDT ?

Reply:
done